test/fixtures/track-basic.json

```diff
@@ -12,7 +12,8 @@
       "query": "analytics"
     },
     "context": {
-      "ip": "10.0.0.1"
+      "ip": "10.0.0.1",
+      "userAgent": "Mozilla/5.0 (PlayBook; U; RIM Tablet OS 1.0.0; en-US) AppleWebKit/534.11 (KHTML, like Gecko) Version/7.1.0.7 Safari/534.11"
```

Comment:
I took this same from the website of the userAgent parser we use, might be worthwhile to update it with the format analytics.js uses.